lnst/Recipes/ENRT/SRIOVNetnsOvSRecipe.py

```diff
@@ -0,0 +1,197 @@
+    |  |    +----+    |  |   |  |    +----+    |  |
+    |  |       |      |  |   |  |       |      |  |
+    |  |vf_representor|  |   |  |vf_representor|  |
+    |  |              |  |   |  |              |  |
+    |  +--OvS bridge--+  |   |  +--OvS bridge--+  |
+    |         |          |   |         |          |
+    |    +-namespace-+   |   |    +-namespace-+   |
+    |   |    vf0     |   |   |   |    vf0     |   |
+    |   +-----------+    |   |   +-----------+    |
+    |      host1         |   |       host2        |
+    +--------------------+   +--------------------+
+
+    All sub configurations are included via Mixin classes.
+
+    The actual test machinery is implemented in the :any:`BaseEnrtRecipe` class.
+    """
+    host1 = HostReq()
+    host1.eth0 = DeviceReq(label="net1", driver=RecipeParam("driver"))
+
+    host2 = HostReq()
+    host2.eth0 = DeviceReq(label="net1", driver=RecipeParam("driver"))
+
+    offload_combinations = Param(default=(
+        dict(gro="on", gso="on", tso="on", tx="on", rx="on"),
+        dict(gro="off", gso="on", tso="on", tx="on", rx="on"),
+        dict(gro="on", gso="off", tso="off", tx="on", rx="on"),
+        dict(gro="on", gso="on", tso="off", tx="off", rx="on"),
+        dict(gro="on", gso="on", tso="on", tx="on", rx="off")))
+
+    def test_wide_configuration(self):
+        """
+        Test wide configuration for this recipe involves switching the device to switchdev
+        mode, adding single VF and mapping the VF, as well as its representor.
+        New namespace is created to mimic container networking, where the VF is moved.
+        Next, VF is assigned an IPv4 and IPv6 address on both hosts.
+        Finally, new OvS bridge is created, where PF and VF representor are added, to
+        ensure correct switching of network traffic between the hosts.
+
+        host1.eth0 = 192.168.101.1/24 and fc00::1/64
+
+        host2.eth0 = 192.168.101.2/24 and fc00::2/64
+        """
+        host1, host2 = self.matched.host1, self.matched.host2
+        configuration = super().test_wide_configuration()
+        configuration.test_wide_devices = []
+
+        for i, host in enumerate([host1, host2]):
+            host.run("systemctl enable openvswitch")
+            host.run("systemctl start openvswitch")
+            host.run("ovs-vsctl set Open_vSwitch . other_config:hw-offload=true")
+
+            host.run(f"devlink dev eswitch set pci/{host.eth0.bus_info} mode switchdev; sleep 2")
+            host.run(f"echo 1 > /sys/class/net/{host.eth0.name}/device/sriov_numvfs; sleep 5")
+
+            vf_ifname = dict(ifname=f"{host.eth0.name}v0")
+            host.map_device("vf_eth0", vf_ifname)
+
+            host.newns = NetNamespace(f"lnst{i + 1}")
+            host.newns.vf_eth0 = host.vf_eth0
+
+            vf_representor_ifname = dict(ifname="eth0")
+            host.map_device("vf_representor_eth0", vf_representor_ifname)
+            host.vf_representor = host.vf_representor_eth0
```

Comment:
Is this just for the purpose of shortening the name? If so, I think it is confusing. Anyone reading this would think why this is needed?

Reply:
is this removed? i still see the `host.vf_representor = host.vf_representor_eth0` in here but i'm not sure if that was the original code that was being commented on.

Reply:
I wasn't testing OvS recipe and forgot about the changes there. Going to fix it now.